src/components/Moon.js

```diff
@@ -1,12 +1,21 @@
-import React from 'react'
-import moon from '../assets/moon.png'
-import { nanoid } from 'nanoid'
+import React, { useEffect } from 'react'
 
+import moon from '../assets/images/moon.png'
+import { useState } from 'react'
+import { useLocation } from 'react-router-dom'
 
-export const Moon = () => { 
-  const id = nanoid(10)
-  return (
-    <img alt="moon" src={moon} key={id} className="moon"/>
-  )
-}
+export const Moon = ({ id }) => {
+	const [isInactive, setInactive] = useState(false)
+	const location = useLocation().pathname.split('/')[2]
+
+	useEffect(
+		() => {
+			if (location <= id && !isInactive) {
+				setInactive(true)
+			}
+		},
+		[id, isInactive, location]
+	)
 
+	return <img alt='moon' src={moon} key={'moon'} className={`moon ${isInactive && 'inactive'}`} />
```

Comment:
`moon ${false && 'inactive'}` evaluates to 'moon false' and
`moon ${true && 'inactive'}` evaluates to 'moon inactive'

is that what you were looking for?
based off my previous suggestion you could do something like `moon ${state.active ? class1 : class2}`

Reply:
Hahah. Using a double negative like inactive and false is too confusing I am unsure why I did it that way. I am so thankful for this code review lol